R-package/R/utils.R

```diff
@@ -59,53 +57,6 @@ lgb.last_error <- function() {
 
 }
 
-lgb.call <- function(fun_name, ret, ...) {
```

Comment:
These `lgb.call()` functions can be removed because once routine registration is used, referencing routines using strings does not work: https://github.com/microsoft/LightGBM/pull/4155#issuecomment-812315498.